### PR TITLE
Fix math

### PR DESCRIPTION
**Pull request by aaronc**

### CODE REVIEW

File: account.go [hunk 1]
```diff
@@ -65,17 +92,19 @@ func RecordToAccount(rec Record, genesisTime time.Time) (Account, error) {
 		}, nil
 	}
 
+	// calculate dust, which represents an uregen-integral remainder, represented as an apd.Decimal of regen
```
Comment (clevinson) [inline account.go]:
```suggestion
	// calculate dust, which represents an uregen-integral remainder, represented as a decimal value of regen
```
Removing apd.Decimal from comment

File: account.go [hunk 2]
```diff
@@ -153,18 +188,15 @@ const (
...
 )
 
-var tenE6 *apd.Decimal
+var tenE6 Dec = NewDecFromInt64(1000000)
 
-func init() {
-	var err error
-	tenE6, _, err = apd.NewFromString("1000000")
+func ToCosmosAccount(acc Account, genesisTime time.Time) (auth.AccountI, *bank.Balance, error) {
+	err := acc.Validate()
```
Comment (clevinson) [inline account.go:194]: 🙏 nice!

File: testutil.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+)
+
+func RequireDecEqual(t *testing.T, expected, actual Dec, msgAndArgs ...interface{}) {
+	if !expected.IsEqual(actual) {
+		assert.Fail(t, fmt.Sprintf("%s != %s", expected, actual), msgAndArgs...)
+		t.FailNow()
+	}
+}
+
+func RequireAccountEqual(t *testing.T, expected, actual Account) {
```
Comment (clevinson) [inline testutil.go:19]: Looks nice. Can't fid any usages of this here, will we be using it in later PRs?

**Review by clevinson — COMMENTED** [review]
One comment tweak (we're no longer using apd), otherwise lgtm 👍

**Review by clevinson — APPROVED** [review]
ack

File: testutil.go (hunk 3 above)
Comment (aaronc) [reply testutil.go:19]: Yes

**Pull request merged**